src/app/getNavItems.ts

```diff
@@ -43,33 +40,10 @@ export function getNavItems(isMultizoneMode: boolean, hasMeshes: boolean): NavIt
       anchorRouteName: 'zone-egress-abstract-view',
       insightsFieldAccessor: 'global.ZoneEgress',
     },
-  ]
-
-  const meshItems: NavItem[] = !hasMeshes
-    ? []
-    : [
-      {
-        name: 'Mesh',
-        categoryTier: 'primary',
-      },
-      {
-        name: 'Mesh selector',
-        isMeshSelector: true,
-      },
-      {
-        name: 'Overview',
-        routeName: 'mesh-overview-view',
-        anchorRouteName: 'mesh-detail-view',
-        usesMeshParam: true,
-      },
-    ]
-
-  return [
     {
-      name: 'Home',
-      routeName: 'home',
+      name: 'Meshes',
+      routeName: 'mesh-list-view',
+      anchorRouteName: 'mesh-abstract-view',
```

Comment:
Just a comment/thought. I kinda think we could do away with `getNavItems` entirely and just configure the Sidebar in the sidebar itself, feels even more indirection-y now we barely have anything in the sidebar. Just to be clear no action here just thinking out loud.